announcements.py

```diff
@@ -36,32 +50,37 @@ def run(self):
             time.sleep(sleep_time_seconds + 1) # Offset by 1 second
 
             # POST Request to send message
-            # url = 'https://your.endpoint.here'
+            current_day = datetime.datetime.now().strftime('%A').lower()
+            print("[" + str(datetime.datetime.now()) + "] DEBUG - Announcement (id: " + str(self.id) + ") post on " + current_day + ": " + str(self.selected_days[current_day]))
```

Comment:
Minor nitpick but consider using formatted string literals for cases like this in python. It will make it look cleaner :)

Refer to the following:

- https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python